doc/config.rst

```diff
@@ -1698,13 +1698,21 @@ These options influence LaTeX output. See further :doc:`latex`.
 
         .. versionadded:: 1.5
      ``'figure_align'``
-        Latex figure float alignment, default 'htbp' (here, top, bottom, page).
+        LaTeX figure float alignment, default 'htbp' (here, top, bottom, page).
```

Comment:
I think `float alignment` creates confusion for LaTeX user. It should be `float position` (or `location` as used in the Lamport book).